muspinsim_config/test-data/test_7.in

```diff
@@ -0,0 +1,19 @@
+
+#######################################################
+# Muspinsim Input File
+# Generated using Muon Galaxy Tool Muspinsim_Input
+#######################################################
+
+name
+    test_6
+spins
+    mu H
+zeeman 2
+    2 0 0
+zeeman 1
+    1 0 0
+average_axes
+    orientation
+    temperature
+experiment
+    zero_field
```

Comment:
```suggestion
    custom
```
Would also hope to see sections for `x_axis` and `y_axis` once we're testing `custom`, but not sure of the order to expect. Probably easiest to just regenerate the file to match the test output as long is it's as expected.